test/s3_bucket_creation_test.go

```diff
@@ -32,12 +33,18 @@ func TestS3Creation(t *testing.T) {
 	bucketAES256 := terraform.Output(t, terraformOptions, "bucket_aes256")
 	assert.Regexp(t, regexp.MustCompile(`AES256`), bucketAES256)
 
-	assert.Regexp(t, regexp.MustCompile(`^arn:aws:s3:::s3-bucket-*`), bucketArn)
+	assert.Regexp(t, regexp.MustCompile(`arn:aws:s3:::unit-test-bucket*`), bucketArn)
 	// Verify that our Bucket has a policy attached
 	aws.AssertS3BucketPolicyExists(t, awsRegion, bucketID)
 
 	// Verify that our Bucket has versioning enabled
 	actualStatus := aws.GetS3BucketVersioning(t, awsRegion, bucketID)
 	expectedStatus := "Enabled"
 	assert.Equal(t, expectedStatus, actualStatus)
+
+	// Verify bucket notification is created
```

Comment:
Is this needed?

Reply:
Possibly not. I need to check with David for his opinion on whether this test is required. Forgot about that bit.

Reply:
Yes. If there's a way to test the creation of a notification, then we should check that it creates successfully.